bin/populate_env

```diff
@@ -30,15 +30,15 @@ template='{
       "ga_property": ""
       }
     },
-    "hidden_support_section": %HIDDEN_SUPPORT_SECTION%
+    "hidden_support_sections": %HIDDEN_SUPPORT_SECTIONS%
```

Comment:
Pursuant to my comment on the helm chart PR, I think we should expect this env var to be a comma-separated string, and we should parse it into an array in this script

Reply:
(related comment: https://github.com/helxplatform/ui-chart/pull/9#discussion_r752586364)

Reply:
Ahh I see, we need to split the string into an array in bash, does the following syntax look good? 
`hidden_support_sections=(${REACT_APP_HIDDEN_SUPPORT_SECTIONS//,/ })`
First we replace all comma with space and the surrounding parenthesis will covert that into array?

Reply:
I think there's an extra step you need, since we have to go from `"foo,bar"` to `["foo","bar"]` so we need to wrap each individual string in double-quotes to make it valid JSON

Reply:
I'm not super familiar with the syntax, would the following work?

```
for i in "${!hidden_support_sections[@]}"
do 
    hidden_support_sections[$i]="${hidden_support_sections[$i]}"
done
```

Reply:
Maybe, but I'm also thinking piecing it together in bash might not be the ideal approach. Now that I think about it a little more, it might be easier to leave it as a comma-separate string in the env.json, and convert it into an array in the Javascript code when you load the environment params into the window. Would that be easier to implement?

Reply:
yeah, actually I was thinking about that too. This is a little bit unnecessary. UI can get the string can just split it there.